sdk/cosmosdb/azure_data_cosmos/Cargo.toml

```diff
@@ -0,0 +1,43 @@
+[package]
+name = "azure_data_cosmos"
+version = "0.19.0"
+description = "Rust wrappers around Microsoft Azure REST APIs - Azure Cosmos DB"
+readme = "README.md"
+authors.workspace = true
+edition.workspace = true
+license.workspace = true
+repository.workspace = true
+rust-version.workspace = true
+homepage = "https://github.com/azure/azure-sdk-for-rust"
+documentation = "https://docs.rs/azure_data_cosmos"
+keywords = ["sdk", "azure", "rest", "cloud", "cosmosdb", "database"]
+categories = ["api-bindings"]
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+async-trait.workspace = true
+azure_core.workspace = true
+typespec_client_core = { workspace = true, features = ["derive"] }
+time.workspace = true
+tracing.workspace = true
+url.workspace = true
+serde.workspace = true
+
+[dev-dependencies]
+tokio.workspace = true
+serde_json.workspace = true
+azure_identity.workspace = true
+clap.workspace = true
+
+[lints]
+workspace = true
+
+[features]
+default = ["hmac_rust"]
+hmac_rust = ["azure_core/hmac_rust"]
+hmac_openssl = ["azure_core/hmac_openssl"]
+key-auth = [] # Enables support for key-based authentication (Primary Keys and Resource Tokens)
```

Comment:
Use underscores for feature names. New guideline I added yesterday so everything from crate names to features to module names are uniformly underscores.